src/App.jsx

```diff
@@ -16,48 +16,107 @@ function App() {
+            <label
+              htmlFor="ssid"
+              className="block text-sm font-bold text-gray-700"
+            >
               Wi-Fi Name
             </label>
             <input
               type="text"
               id="ssid"
               ref={ssidRef}
               onChange={handleWifiChange}
-              className="border-2 text-xl p-3 lg:text-5xl lg:p-6"
-            ></input>
+              className="mt-1 w-full rounded-md border-gray-200 bg-white text-sm text-gray-700 shadow-sm"
+            />
           </div>
-          <div className="flex flex-col space-y-2">
-            <label htmlFor="password" className="text-lg font-bold">
+          <div className="w-full">
+            <label
+              htmlFor="password"
+              className="block text-sm font-bold text-gray-700"
+            >
               Password
             </label>
-
             <input
               type="text"
               id="password"
               ref={passwordRef}
               onChange={handleWifiChange}
-              className="border-2 text-xl p-3 lg:text-5xl lg:p-6"
-            ></input>
+              className="mt-1 w-full rounded-md border-gray-200 bg-white text-sm text-gray-700 shadow-sm"
+            />
+          </div>
+        </form>
+      </div>
+      <div className="space-y-2">
+        {/* I don't control this width with TailwindCSS -- it's just 320px wide */}
+        <div>
+          <label className="block text-sm font-bold text-gray-700">
+            QR Code
+          </label>
+
+          <div className="mt-1 border w-full rounded-md border-gray-200 bg-white text-sm text-gray-700 shadow-sm">
+            <QRCode
+              data={wifiString}
+              options={{ width: 320 }}
+              ref={canvasRef}
+            ></QRCode>
           </div>
         </div>
-      </form>
+        <div className="space-y-2">
+          <button
+            type="button"
+            onClick={(e) => {
+              e.preventDefault();
+              downloadCanvas(
+                canvasRef.current,
+                `${wifi.ssid ? wifi.ssid : "wifi"}-qr-code.png`,
+              );
+            }}
+            className="mt-2 w-full text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center inline-flex items-center me-2 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800"
+          >
+            {/* https://remixicon.com/ */}
+            <svg
+              className="w-3.5 h-3.5 me-2"
+              aria-hidden="true"
+              xmlns="http://www.w3.org/2000/svg"
+              fill="currentColor"
+              viewBox="0 0 24 24"
+            >
+              <path d="M13 10H18L12 16L6 10H11V3H13V10ZM4 19H20V12H22V20C22 20.5523 21.5523 21 21 21H3C2.44772 21 2 20.5523 2 20V12H4V19Z"></path>
+            </svg>
+            Download
```

Comment:
Change this to `Download as PNG`

Reply:
Maybe `Save Image as PNG`? idk, probably doesn't matter much. User testing will reveal way more about this than me thinking real hard about it on my own